src/onegov/org/app.py

```diff
@@ -193,23 +193,25 @@ def ticket_permissions(self):
         return result
 
     @orm_cached(policy='on-table-change:pages')
-    def homepage_pages(self) -> dict[int, list[Topic]]:
-
+    def topics_cached(self) -> list[tuple[int, 'Topic']]:
         def visit_topics(
-            pages: 'Iterable[Page]',
-            root_id: int | None = None
+            pages: 'Iterable[Page]', root_id: int | None = None
```

Comment:
Just a small style nit, but when the whole signature doesn't fit on one line, I would always prefer if each argument had its own line.